manifests/install.pp

```diff
@@ -7,9 +7,12 @@
 # [package_file_name]
 #   The puppet-agent package file name.
 #   (see puppet_agent::prepare::package_file_name)
+# [version]
+#   The puppet-agent version to install.
 #
 class puppet_agent::install(
   $package_file_name = undef,
+  $package_version   = 'present',
```

Comment:
Will this ever be used? We always pass an argument in init.pp, so seems like it will either be a version or undef.